Polyhedron/demo/Polyhedron/Plugins/PMP/Point_inside_polyhedron_widget.ui

```diff
@@ -87,6 +87,19 @@
       </layout>
      </widget>
     </item>
+    <item>
+     <spacer name="verticalSpacer">
+      <property name="orientation">
+       <enum>Qt::Vertical</enum>
+      </property>
+      <property name="sizeHint" stdset="0">
+       <size>
+        <width>20</width>
+        <height>40</height>
```

Comment:
I understand the need for a space, when there is a lot of space, so that the note and the text are at the top of the Window. But, when the dockwidget is reduced as much as possible, why let the space take a minimal height of 20 ?